packages/crypto/src/transactions/types/transaction.ts

```diff
@@ -97,4 +97,44 @@ export abstract class Transaction implements ITransaction {
     public hasVendorField(): boolean {
         return false;
     }
+
+    /**
+     * Calculate the expiration height of a transaction.
+     * An expiration height H means that the transaction cannot be included in block at height
+     * H or any higher block.
+     * If the user did not specify an expiration height when creating the transaction then
+     * we calculate one from the timestamp of the transaction creation and the configured
+     * maximum transaction age.
```

Comment:
Append `...age (for v1 transactions). v2 transactions have a dedicated expiration property`.